source/includes/commit-process.rst

```diff
@@ -0,0 +1,138 @@
+.. step:: Verify the status of mongosync.
+
+   Call the :ref:`progress <c2c-api-progress>` endpoint to determine
+   the status of ``mongosync`` before starting the cutover process.
+   Ensure that the ``mongosync`` process status indicates the
+   following values:
+
+   - ``canCommit`` is ``true``.
+
+   - ``lagTimeSeconds`` is small (near ``0``).
+
+      If ``lagTimeSeconds`` isn't close to ``0`` when the cutover
+      starts, cutover might take a long time.
+
+   The following example returns the status of the synchronization process.
+
+   Request
+   ~~~~~~~
+
+   .. literalinclude:: /includes/api/requests/progress.sh
+      :language: shell
+
+   Response
+   ~~~~~~~~
+
+   .. literalinclude:: /includes/api/responses/progress.json
+      :language: json
+      :copyable: false
+      :emphasize-lines: 5, 8
+
+.. step:: Stop any write operations to the synced collections on the source.
```

Comment:
Can we clarify how this tutorial compares with the Finalize Cutover Process tutorial? It'd be helpful given the overlap between these two

Reply:
just to add a bit more info - the major differences are:
- reverse requires write blocking
- reverse does not work with filtered sync

besides that, steps 1-7 should be the exactly same

Reply:
If we move forward with linking to the cutover process, I will add these considerations to the reverse sync page